vue-app/src/components/WalletWidget.vue

```diff
@@ -1,11 +1,7 @@
 <template>
   <div :class="{ container: !isActionButton }">
-    <template v-if="!isLoaded()"></template>
-    <div v-if="walletProvider && !isCorrectNetwork()" class="provider-error">
-      Please change network to {{ networkName }}
-    </div>
```

Comment:
We are not going to face this scenario anymore. You will be connected or disconnected.

As mentioned in the description, if the user changes the network, the app is going to automatically disconnect him.